src/app/cdk/platform-info/platform-info.service.ts

```diff
@@ -213,9 +213,16 @@ export class PlatformInfoService {
     this.platform = {
       ...this.platform,
       hasOauthParameters: this.hasOauthParameters(),
-      social: this.window.location.href.toLowerCase().indexOf('social') >= 0,
+      social:
+        this.window.location.pathname.toLowerCase().indexOf('social-linking') >=
+        0,
       institutional:
-        this.window.location.href.toLowerCase().indexOf('institutional') >= 0,
+        this.window.location.pathname
+          .toLowerCase()
+          .indexOf('institutional-linking') >= 0 ||
+        this.window.location.pathname
+          .toLowerCase()
+          .indexOf('institutional-signin') >= 0,
```

Comment:
Nice fix! 
Maybe we should also use window.location.pathname here as well 

https://github.com/ORCID/orcid-angular/blob/bde83b34b66386012920018cc445e0573c83e1d8/src/app/cdk/platform-info/platform-info.service.ts#L179

So the app don't get break even if the user sends `social-linking` at any point on the query parameters

Reply:
Done!